docs/api.rst

```diff
@@ -1,10 +1,10 @@
 API docs
 --------
 
-.. autoclass:: openfe.setup.Molecule
+.. autoclass:: openfe.setup.LigandMolecule
```

Comment:
> Isn't saying "use ABFE for this node" is similar to saying "use SAMS for this edge"?

RBFE != ABFE, they involve different setup processes and they definitely don't need you to have edges. I know it's not what you're trying to say, but I'd be careful not to conflate using ABFE with using a specific Hamiltonian sampling strategy, especially since you can have both ABFE SAMS and RBFE SAMS (same for REPEX, etc...).

You could imagine them using the same network as what we do for RBFEs (which is something I remember bringing up before on slack), but our current networks are bought into edges and can't have standalone nodes whilst an ABFENetwork could just be a collection of unconnected nodes, that _may_ grow to add edges in the future.

My suggestion here I think reflects what we last discussed, which is to have a collection of ABFE nodes being added to your "simulation network" (or "simulation plan") if you wish.

I _think_ we need to discuss the scope of "Network" - are we going to have other types of networks or are we limiting it to RBFEs? etc...

Reply:
> but our current networks are bought into edges and can't have standalone nodes

Actually, after our Slack discussion on this, I made sure that we *can* have standalone nodes. Originally that gave an error (because I thought the network was RBFE-only) but you convinced that we should allow disconnected nodes. There are even tests to ensure it! 😊

Those nodes are the same class as our other nodes; I did this with the idea that you can assign calculation method to them later, in the same way that you can assign edge calculation methods later.

Reply:
I think for the next step, we'll use a different Network object.  For planning RBFE we only considered a subset of information (no protein, optional coordinates, no atom types), and rather than try and layer this information onto the existing object, I'd rather we implement a way to turn this `LigandNetwork` into the bigger object we need.

The different Network object is annoying but this has the advatantage that `LigandNetwork` is a clean way to get on/off the openfe ecosystem:
- You could plan your ligand mappings elsewhere, and only have to convert this to `LigandNetwork` to join in
- You could only plan your ligand mappings with us and take this lightweight object elsewhere

I think we might end up recreating `Network` in `utils/` (and LigandNetwork subclasses this) if the simulation Network wants to have similar `to_graphml` like behaviour.